wgengine/magicsock/relaymanager.go

```diff
@@ -870,7 +870,11 @@ func doAllocate(ctx context.Context, server netip.AddrPort, discoKeys [2]key.Dis
 	if err != nil {
 		return udprelay.ServerEndpoint{}, err
 	}
-	resp, err := http.DefaultClient.Do(req)
+	transport := &http.Transport{
+		DisableKeepAlives: true, // this transport is meant to be used once
+	}
```

Comment:
Does this even work on Apple platforms? It's not using the ioctl to let you dial within the sandbox to peers.

IIRC you need to use `PeerAPITransport` or similar.